resources/views/app/includes/contents/single_episode.blade.php

```diff
@@ -1,18 +1,17 @@
 <div class="row">
 
 <!-- Side Nav -->
+
     <div class="col s3">
         <div class="hide-on-small-only">
             <div class="collection">
-                <a href="#" class="collection-item">Channels <span class="new badge">4</span></a>
-                <a href="#" class="collection-item">Favourites <span class="new badge">0</span></a>
-                <a href="#" class="collection-item" id="view-all-episodes">See all episodes
-                    <span class="badge">10+</span>
-                </a>
-                @can('guest', Auth::check())
-                <a href="{{ URL::to('about') }}" class="collection-item">About</a>
-                <a href="{{ URL::to('privacypolicy') }}" class="collection-item">Privacy Policy</a>
-                <a href="{{ URL::to('faqs') }}" class="collection-item">FAQs</a>
+                <a href="/channels" class="collection-item">Channels <span class="new badge">4</span></a>
+                <a href="/episodes" class="collection-item" id="view-all-episodes">See all episodes
+                    <span class="badge">10+</span></a>
+                    @can('guest', Auth::check())
+                    <a href="{{ URL::to('about') }}" class="collection-item">About</a>
+                    <a href="{{ URL::to('privacypolicy') }}" class="collection-item">Privacy Policy</a>
+                    <a href="{{ URL::to('faqs') }}" class="collection-item">FAQs</a>
```

Comment:
**Code Quality - Nitpick**

As a sidenav, it should be moved into a partial file. Then, the partial file should be included here.